src/__tests__/index.js

```diff
@@ -51,4 +51,12 @@ describe('#SitemapGenerator', () => {
     expect(data.lastMod).toBe(queueItem.stateData.headers['last-modified']);
     expect(data.formattedLastMod).toBe('2023-01-05');
   });
+
+  test('::parsePage should respect the ignoreCanonacalized option', () => {
+    const page =
```

Comment:
also test cases
- where ignore option is on, canonical exists, canonical matches
- where ignore option is on, canonical is missing

Reply:
Tests updated here: https://github.com/zendesk/sitemap-generator/pull/7/commits/bb63727954b7deafad81e7ab31ae07f5af44a556